test/system/irohad_test.cpp

```diff
@@ -122,9 +128,26 @@ DROP TABLE IF EXISTS index_by_id_height_asset;
 
  public:
   boost::filesystem::path irohad_executable;
-  std::chrono::milliseconds timeout = 1s;
+  const std::chrono::milliseconds kTimeout = std::chrono::seconds(1);
+  const std::string kAddress;
+  const uint16_t kPort;
 
- private:
+  boost::optional<child> iroha_process_;
```

Comment:
What the point to make it optional?

Reply:
The optional reflects the possibility of missing initialized Iroha. The field `iroha_process_` cannot be just the class member because it will require initialization in fixture constructor's initializer list. We cannot do it because the initialization parameters for `iroha_process_` are not known by the moment of the fixture construction.